astropy/time/tests/test_methods.py

```diff
@@ -0,0 +1,199 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+# TEST_UNICODE_LITERALS
+import numpy as np
+
+from .. import Time
+from ...utils.compat.numpycompat import NUMPY_LT_1_9
+
+class TestManipulation():
+    """Manipulation of Time objects, ensuring attributes are done correctly."""
+
+    def setup(self):
+        mjd = np.arange(50000, 50010)
+        frac = np.arange(0., 0.999, 0.2)
+        self.t0 = Time(mjd[:, np.newaxis] + frac, format='mjd', scale='utc')
+        self.t1 = Time(mjd[:, np.newaxis] + frac, format='mjd', scale='utc',
+                       location=('45d', '50d'))
+        self.t2 = Time(mjd[:, np.newaxis] + frac, format='mjd', scale='utc',
+                       location=(np.arange(len(frac)), np.arange(len(frac))))
+
+    def test_ravel(self):
+        t0_ravel = self.t0.ravel()
+        assert t0_ravel.shape == (self.t0.size,)
+        assert np.all(t0_ravel.jd1 == self.t0.jd1.ravel())
+        assert np.may_share_memory(t0_ravel.jd1, self.t0.jd1)
```

Comment:
It doesn't need repeating in every test, but at least once or twice you should also check that `jd2` is doing the right thing.